runners/apex/src/main/java/org/apache/beam/runners/apex/translation/operators/ApexGroupByKeyOperator.java

```diff
@@ -106,7 +106,7 @@ public void process(ApexStreamTuple<WindowedValue<KV<K, V>>> t) {
         }
         processElement(t.getValue());
       } catch (Exception e) {
-        Throwables.propagateIfPossible(e);
+        Throwables.throwIfUnchecked(e);
```

Comment:
Can 'throw new RuntimeException' be folded into throwIfUnchecked(e) ?
That way, code is simplified on the caller side.